micro-ui/web/micro-ui-internals/packages/react-components/src/atoms/svgindex.js

```
@@ -1,6 +1,26 @@
 
+const PopulationNew = ({ className, width = "40", height = "46", style = {}, fill = COLOR_FILL, onClick = null }) => (
+  <svg
+    width={width}
+    height={height}
+    className={className}
+    onClick={onClick}
+    style={style}
+    viewBox="0 0 32 27"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
```

Biome

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)